gensound/transforms.py

```diff
@@ -187,34 +187,49 @@ def realise(self, audio):
 
 ######### Level/ampltidue Stuff ###################
 
-class Fade(Transform):
-    min_fade = -50
-    
-    def __init__(self, is_in=True, duration=3e3):
-        self.is_in = is_in
+class Fade(Transform):   
+    def __init__(self, duration=None, curve=None, is_in=True, degree=1):
```

Comment:
argument `is_in` is in my opinion more critical than the others, it should appear first.